extension/src/features/input_password_modal.rs

```diff
@@ -130,16 +132,27 @@ pub fn input_password_modal(props: &Props) -> Html {
         PasswordFor::UnlockingApp => "Input your password to unlock extension",
     };
 
+    let onkeypress = {
+        let onclick = Rc::new(onclick.clone());
+        Callback::from(move |event: KeyboardEvent| {
+            if event.key() == "Enter" {
+                let _ = MouseEvent::new("click").map(|e| {
+                    let _ = &onclick.emit(e);
+                });
+            }
```

Comment:
I don't really care very much about the error in this case, since it just prevents the user from pressing enter to accept the password, and I it wouldn't be much help displaying the error either. The user can just click the Accept button. I could log it to the developer tools terminal, what do you think?

Reply:
no needed! lgtm!